js/assessments/wordComplexityAssessment.js

```diff
@@ -54,11 +90,8 @@ var calculateComplexity = function( wordCount, tooComplexWords, recommendedValue
 var wordComplexityAssessment = function( paper, researcher, i18n ) {
 	var wordComplexity = researcher.getResearch( "wordComplexity" );
 	var wordCount = wordComplexity.length;
-	var recommendedValue = 4;
-	var tooComplexWords = filter( wordComplexity, function( syllables ) {
-		return( syllables >= recommendedValue );
-	} ).length;
-	var complexityResult = calculateComplexity( wordCount, tooComplexWords, recommendedValue, i18n );
+	var tooComplexWords = filterComplexity( wordComplexity ).length;
```

Comment:
Considering this isn't being used elsewhere, you could call this line within `calculateComplexity` instead of here.